### DISCUSSION TITLE
uvicorn reload-dir doesn't work on openBSD

### DISCUSSION THREAD

**Discussion by sandeep-gh** [Kludex/uvicorn#1227] [Potential Issue]
What are the requirements for reload to work? On OpenBSD I get warning that current cornfiguration will not reload as not all conditions are met. It doesn't give too much info on what aspect of reload failed. Can reload only work on certain file systems?

**Comment by Kludex** [comment]
What's the error message?

**Reply by sandeep-gh** [reply]
There is no error message. Just one warning:
```
WARNING:  Current configuration will not reload as not all conditions are met,please refer to documentation.
```

**Reply by Kludex** [reply]
```python
        if (
            reload_dirs or reload_includes or reload_excludes
        ) and not self.should_reload:
            logger.warning(
                "Current configuration will not reload as not all conditions are met,"
                "please refer to documentation."
            )
```
That condition is not met... But I'll be willing to accept a PR to improve that warning.
